UserInterface/SwitchDemos/SwitchDemos/SwitchDemos/MainPage.xaml.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.ComponentModel;
+using System.Windows.Input;
+using Xamarin.Forms;
+
+namespace SwitchDemos
+{
+    // Learn more about making custom code visible in the Xamarin.Forms previewer
+    // by visiting https://aka.ms/xamarinforms-previewer
+    [DesignTimeVisible(false)]
```

Comment:
Remove this attribute (and comments). I know the templates add it, but it's not required for this page.